src/pineko/cli/kfactor.py

```diff
@@ -0,0 +1,41 @@
+"""CLI entry point to generation of the inclusion of kfactor in a grid."""
+
+import pathlib
+
+import click
+import pineappl
+import rich
+
+from .. import kfactor
+from ._base import command
+
+
+@command.command("kfactor")
+@click.argument("grids_folder", type=click.Path(exists=True))
+@click.argument("kfactor_folder", type=click.Path(exists=True))
+@click.argument("yamldb_path", type=click.Path(exists=True))
+@click.argument("compound_folder", type=click.Path(exists=True))
+@click.argument("target_folder", type=click.Path(exists=False))
```

Comment:
```suggestion
@click.argument("target_folder", type=click.Path(exists=True))
```

Or alternatviely it would make sense for the script to create the folder, e.g. by calling `mkdir` on the `pahtlib.Path` below.